shapiq/exact.py

```diff
@@ -830,6 +834,32 @@ def probabilistic_value(self, index: str, *args, **kwargs) -> InteractionValues:
         self._computed[(index, order)] = probabilistic_value
         return copy.copy(probabilistic_value)
 
+    def compute_egalitarian_least_core(self, index: str, *args, **kwargs):
+        order = 1
+        if index == "ELC":
+            # Check for normalized game
+            if self.baseline_value != 0:
+                raise ValueError(
+                    "The egalitarian least-core can only be computed on normalized games!"
+                    "Please set game_fun.normalize=True!"
+                )
+
+            # Compute egalitarian least-core
+            egalitarian_vector, subsidy = core.egalitarian_least_core(
+                n_players=self.n,
+                game_values=self.game_values,
+                coalition_lookup=self.coalition_lookup,
+                grand_coalition_tuple=self._grand_coalition_tuple,
+            )
+        else:
+            raise ValueError(f"Index {index} not supported")
```

Comment:
I don't like this if/else structure here at the moment. Let me elaborate:

1. First, I don't like the check with "ELC" as only ELC is acceptable here, any other index will lead to an error. The user should not have to specify any `index `here. So we could just drop the `index `parameter entirely and leave args and kwargs as they are there now.
2. Second, I am not 100% sure if we cannot do something if `if self.baseline_value != 0:`. Can we re-normalize here? If not then we need to error out, yes. But if yes, I would like to do this here.
3. Third, the "Please set game_fun.normalize=True!" is a bit misleading. If a `game_fun` is an instance of `Game` then this will not resolve the problem. Setting the attribute will not trigger a re-normalization of the game. The normalize parameter must be set to `True` at the initialization of the game. So we would at least update the error text here to reflect that. :)